jina/parsers/orchestrate/runtimes/worker.py

```diff
@@ -39,6 +39,15 @@ def mixin_worker_runtime_parser(parser):
     Dictionary of keyword arguments that will override the `with` configuration in `uses`
     ''',
     )
+    gp.add_argument(
+        '--env-from-secret',
+        action=KVAppendAction,
+        metavar='KEY: VALUE',
+        nargs='*',
+        help='''
+    Dictionary of keyword arguments that will override the `env_from_secret` configuration
```

Comment:
improve description here. Also, the `parser` is not the correct, this should be in the `deployment` mixin